solr/solrj/src/java/org/apache/solr/common/cloud/ReplicaPosition.java

```diff
@@ -19,17 +19,20 @@
 
 
 public class ReplicaPosition implements Comparable<ReplicaPosition> {
```

Comment:
It's not from this PR, but the text `Note: this class has a natural ordering that is inconsistent with equals.` should have been added to the class Javadoc that should have been here.